chia/full_node/full_node_api.py

```diff
@@ -1264,8 +1264,9 @@ async def send_transaction(self, request: wallet_protocol.SendTransaction, *, te
         )
         # Waits for the transaction to go into the mempool, times out after 45 seconds.
         status, error = None, None
-        for i in range(450):
-            await asyncio.sleep(0.1)
+        sleep_time = 0.01
+        for i in range(int(45 / sleep_time)):
+            await asyncio.sleep(sleep_time)
```

Comment:
oh. in an ideal world we wouldn't just blindly sleep here, but we would have a queue for responses that we could wait on.

Reply:
Yes true, but this is out of scope of this PR>=